.travis.yml

```diff
@@ -16,11 +16,10 @@ install:
   - pip install codecov
 
 before_script:
-  - if [[ -z "$NO_COVERAGE" ]]; then COV_OPT='--cov'; fi
```

Comment:
It has been unconditional for a while. I didn't test extensively, but the speed gains without coverage on PyPy seemed very small and not worth the added complexity.